### PR TITLE
fix: serverless/dedicated/clustered support chunked and chunk_size

### PR DESCRIPTION
**Pull request by jeffreyssmith2nd**
Closes https://github.com/influxdata/idpe/issues/18411

_Describe your proposed changes here._

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md [hunk 1]
```diff
@@ -457,7 +457,8 @@ For {{% product-name %}} v1 API `/query` requests, set parameters as listed in t
 
 Parameter | Allowed in | Ignored | Value
 ----------|------------|---------|-------------------------------------------------------------------------
-`chunked` |            | Ignored | N/A _(Note that an unbounded query might return a large amount of data)_
+`chunked` | Query string | Honored | Returns points in streamed batches instead of in a single response. If set to `true`, InfluxDB chunks responses by series or by every 10,000 points, whichever occurs first.
+`chunked_size` | Query string | Honored | **Requires `chunked` to be set as well**. If set to a specific value, InfluxDB chunks responses by series or by that number of points.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md:461]:
```suggestion
`chunked_size` | Query string | Honored | **Requires `chunked` to be set to `true`**. If set to a specific value, InfluxDB chunks responses by series or by this number of points.
```

**Review by jstirnaman — APPROVED** [review]
Looks great! Thanks!

File: content/influxdb/cloud-serverless/query-data/execute-queries/v1-http.md [hunk 2]
```diff
@@ -62,7 +62,8 @@ For {{% product-name %}} v1 API `/query` requests, set parameters as listed in t
 
 Parameter   | Allowed in   | Ignored | Value
 ------------|--------------|---------|-------------------------------------------------------------------------
-`chunked`   |              | Ignored | N/A _(Note that an unbounded query might return a large amount of data)_
+`chunked` | Query string | Honored | Returns points in streamed batches instead of in a single response. If set to `true`, InfluxDB chunks responses by series or by every 10,000 points, whichever occurs first.
+`chunked_size` | Query string | Honored | **Requires `chunked` to be set as well**. If set to a specific value, InfluxDB chunks responses by series or by that number of points.
```
Comment (sanderson) [inline content/influxdb/cloud-serverless/query-data/execute-queries/v1-http.md:66]:
```suggestion
`chunked_size` | Query string | Honored | **Requires `chunked` to be set to `true`**. If set to a specific value, InfluxDB chunks responses by series or by this number of points.
```

File: content/influxdb/clustered/guides/api-compatibility/v1/_index.md [hunk 3]
```diff
@@ -456,7 +456,8 @@ For {{% product-name %}} v1 API `/query` requests, set parameters as listed in t
 
 Parameter | Allowed in | Ignored | Value
 ----------|------------|---------|-------------------------------------------------------------------------
-`chunked` |            | Ignored | N/A _(Note that an unbounded query might return a large amount of data)_
+`chunked` | Query string | Honored | Returns points in streamed batches instead of in a single response. If set to `true`, InfluxDB chunks responses by series or by every 10,000 points, whichever occurs first.
+`chunked_size` | Query string | Honored | **Requires `chunked` to be set as well**. If set to a specific value, InfluxDB chunks responses by series or by that number of points.
```
Comment (sanderson) [inline content/influxdb/clustered/guides/api-compatibility/v1/_index.md:460]:
```suggestion
`chunked_size` | Query string | Honored | **Requires `chunked` to be set to `true`**. If set to a specific value, InfluxDB chunks responses by series or by this number of points.
```

**Review by sanderson — APPROVED** [review]
Just some minor suggestions. Looks good. Thanks @jeffreyssmith2nd!

File: content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md (hunk 1 above)
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md:461]: We also need to add these in API reference docs:
- https://docs.influxdata.com/influxdb/cloud-serverless/api/v1-compatibility
- https://docs.influxdata.com/influxdb/cloud-dedicated/api/v1-compatibility
- https://docs.influxdata.com/influxdb/clustered/api/v1-compatibility

**Pull request merged**